src/_posts/databases/mysql/2000-01-01-backing-up.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Backing Up Your Scalingo for MySQL® Addon
+nav: Backing Up
+modified_at: 2024-03-05 12:00:00
+tags: databases mysql addon
+index: 8
+---
+
+
+Starter and Business plans of Scalingo for MySQL® include automated and
+managed backups so you don't have to worry about them.
+
+We use [On-Demand](#understanding-on-demand-backups) backups to create the
```

Comment:
issue: I don't really like the words "on demand" for these backups. I find this confusing. I would use the word "periodic", like on the DB dashboard.

Reply:
OK for me :smiley:
Fixed in https://github.com/Scalingo/documentation/pull/2501/commits/a09a0db9160463946bf57f30ad1092204161e7b9